src/views/CodeOfConduct.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="font-bebas text-wri-dark-blue">
+    <CodeOfConductHeader></CodeOfConductHeader>
+    <CodeOfConductContent></CodeOfConductContent>
+  </div>
+</template>
+
+<script>
+import CodeOfConductHeader from './CodeOfConduct/CodeOfConductHeader.vue'
+import CodeOfConductContent from './CodeOfConduct/CodeOfConductContent.vue'
+export default {
+  name: 'codeofconduct',
+  components: {
+    CodeOfConductHeader,
+    CodeOfConductContent,
+  },
+}
+</script>
+
+<style>
+header p {
+  @apply text-2xl;
+  @apply font-black;
```

Comment:
```suggestion
  @apply font-bold;
```

I don't think we want `black` on this as that made it fairly unreadable. This may simply be an issue with how the fonts were working before we merged in the big CFP Prop PR yesterday as I didn't have the `black` weight even being pulled from Google Fonts.